### ISSUE TITLE
Mermaid error

### ISSUE THREAD

**Issue by dxcore35** [lynchjames/obsidian-day-planner#8]
## System info

System: Mac OS
Obsidian: Latest
Plugin 0.2.2

## Error

```
Error parsing Mermaid diagram!

Parse error on line 7:
...ew      :10-30, 60mm    Reading      :1
-----------------------^
Expecting 'taskData', got 'NL'
```

## Diagram

```
```mermaid
gantt
    title Day Planner for 7th November 2020
    dateFormat  HH-mm
    axisFormat %H:%M
    section Tasks
    Review notes from yesterday      :09-45, 45mm
    Create new notes for #article review      :10-30, 60mm
    Reading      :12-00, 25mm
    Reading      :12-30, 90mm
    Review notes and update daily note [[20201103]]      :15-00, 45mm
    Walk      :15-45, 45mm
    Reading      :16-30, 50mm
    Prep for tomorrow's meetings      :17-20, 340mm
    END      :23-00, 0mm
    section Breaks
    BREAK      :11-30, 30mm
    BREAK      :12-25, 5mm
    BREAK      :14-00, 60mm
```
```

**Comment by lynchjames** [comment]
Thanks, could be the `#` from `#article` on that line causing the issue. Does removing it render the chart?

I will look at escaping.

**Comment by dxcore35** [comment]
Yes the # is making the problem! Anyway please remove the heading, there is no need to write it down as we see daily note date in the Obsidian header (the name of file), it just occupy space and provide no extra information.

**Comment by lynchjames** [comment]
Fix and update available in latest release 0.2.3.
